src/plugin/GoogleCalendar/EventSyncWindow.cpp

```diff
@@ -182,20 +205,16 @@ EventSyncWindow::_LoadSyncData()
 			_SetStatusLabel(lastSyncStatus, *lastSyncTime);
 		}
 	}
-
 	else
 	{
-		BAlert* alert  = new BAlert(B_TRANSLATE("Error"),
-			B_TRANSLATE("There was an error in loading the last sync data."),
-			NULL, B_TRANSLATE("OK"),NULL, B_WIDTH_AS_USUAL, B_WARNING_ALERT);
+		_SetStatusMessage(B_TRANSLATE("There was an error in loading the last sync data."));
```

Comment:
As the window width is limited (maybe make it resizable?), status messages should be short.

```suggestion
		_SetStatusMessage(B_TRANSLATE("Error loading last sync data."));
```